generators/Generators.fs

```diff
@@ -864,11 +864,22 @@ type PerfectNumbers() =
 
 type PhoneNumber() =
     inherit GeneratorExercise()
+
+    override __.PropertiesWithIdentifier _ = ["expected"]
+
+    override __.IdentifierTypeAnnotation (_, _, _) = Some "Result<uint64,string>"
     
     override __.RenderExpected (_, _, value) =
-        value 
-        |> Option.ofNonNull
-        |> Option.renderParenthesized
+        match value.SelectToken "error" with
+        | null  -> 
+            value
+            |> string
+            |> sprintf "Ok %sUL"
+        | error -> 
+            error
+            |> string
+            |> sprintf "Error \"%s\""
+        |> String.parenthesize
```

Comment:
I think you can drop this line, as the parentheses in the declaration are unneeded I believe.